contrib/aws/aws-sdk-go/aws/example_test.go

```diff
@@ -25,3 +31,33 @@ func Example() {
 		Bucket: aws.String("some-bucket-name"),
 	})
 }
+
+func TestWithErrorCheck(t *testing.T) {
+	testOpts := func(errExist bool, opts ...awstrace.Option) func(t *testing.T) {
+		return func(t *testing.T) {
+			mt := mocktracer.Start()
+			defer mt.Stop()
+
```

Comment:
I believe this test belongs in aws_test.go.

Reply:
Moved.